specification/machinelearningservices/data-plane/Microsoft.MachineLearningServices/preview/2019-05-01/execution.json

```diff
@@ -0,0 +1,1103 @@
+          "description": "Fully specified configuration information for the run. Even when that information\r\nis contained in configuration files within the project folder, the client collapses\r\nit all and inlines it into the run definition when submitting a run."
+        },
+        "attribution": {
+          "type": "string"
+        },
+        "telemetryValues": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          }
+        },
+        "snapshotId": {
+          "format": "uuid",
+          "description": "Snapshots are user project folders that have been uploaded to the cloud for subsequent\r\nexecution. This field is required when executing against cloud-based compute targets\r\nunless the run submission was against the API endpoint that takes a zipped project folder\r\ninline with the request.",
+          "type": "string"
+        },
+        "sourceCodeDataReference": {
+          "$ref": "#/definitions/SourceCodeDataReference",
+          "description": "Specifies the cloud source data to use for the run."
+        },
+        "parentRunId": {
+          "description": "Specifies that the run history entry for this execution should be scoped within\r\nan existing run as a child. Defaults to null, meaning the run has no parent.\r\nThis is intended for first-party service integration, not third-party API users.",
+          "type": "string",
+          "example": "myexperiment_155000000001_0"
+        },
+        "runType": {
+          "description": "Specifies the runsource property for this run. The default value is \"experiment\" if not specified.",
+          "type": "string",
+          "example": "experiment"
+        }
+      }
+    },
+    "RunConfiguration": {
+      "type": "object",
+      "properties": {
+        "script": {
+          "description": "The relative path to the python script file. The file path is relative to the source_directory passed to submit run.",
+          "type": "string",
+          "example": "findsquare.py"
+        },
+        "arguments": {
+          "description": "Command line arguments for the python script file.",
+          "uniqueItems": false,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "example": "[\"234\"]"
+        },
+        "sourceDirectoryDataStore": {
+          "description": "The attribute is used to configure the backing datastore for the project share.",
+          "type": "string",
+          "example": "my-source-store"
+        },
+        "framework": {
+          "description": "The supported frameworks are Python, PySpark, CNTK, TensorFlow, and PyTorch. Use Tensorflow for AmlCompute clusters, and Python for distributed training jobs.",
+          "enum": [
+            "Python",
+            "PySpark",
+            "Cntk",
+            "TensorFlow",
+            "PyTorch",
+            "PySparkInteractive",
+            "R"
```

Comment:
R should not be included on this list; it's an experimental feature only available on the Microsoft tenant.

Reply:
Likewise PySparkInteractive.

Reply:
fix on the way

Reply:
done